Updated remote storage setting and added new settings (#125761)
Azure/azure-content: articles/app-service/reference-app-settings.md

| `WEBSITE_PLATFORM_VERSION` | Read-only. App Service platform version. ||
| `HOME` | Read-only. Path to the home directory (for example, `D:\home` for Windows). ||
| `SERVER_PORT` | Read-only. The port the app should listen to. | |

> | `WEBSITE_WARMUP_PATH` | A relative path to ping to warm up the app, beginning with a slash. The default is `/robots933456.txt`. Whenever the platform starts up a container, the orchestrator makes repeated requests against this endpoint. The platform considers any response from this endpoint as an indication that the container is ready. Once the platform considers the container to be ready, it starts forwarding organic traffic to the newly started container. Unless `WEBSITE_WARMUP_STATUSES` is configured, the platform will consider any response from the container at this endpoint - even error codes such as 404 or 502 - as an indication that the container is ready.`NOTE: This appsetting doesn't change the path used by AlwaysOn.` ||

Reviewer: ```suggestion
| `WEBSITE_WARMUP_PATH` | A relative path to ping to warm up the app, beginning with a slash. The default is `/robots933456.txt`. Whenever the platform starts up a container, the orchestrator makes repeated requests against this endpoint. The platform considers any response from this endpoint as an indication that the container is ready. Once the platform considers the container to be ready, it starts forwarding organic traffic to the newly started container. Unless `WEBSITE_WARMUP_STATUSES` is configured, the platform will consider any response from the container at this endpoint - even error codes such as 404 or 502 - as an indication that the container is ready. Note that this appsetting doesn't change the path used by AlwaysOn. ||
```

> | `WEBSITE_WARMUP_STATUSES` | A comma-delimited list of HTTP status codes that will be considered successful when the platform makes warmup pings against a newly started container. Used in conjunction with `WEBSITE_WARMUP_PATH`. By default, any status code is considered an indication that the container is ready for organic traffic. This appsetting can be used to require a specific response before organic traffic is routed to the container. `Example: 200,202`. If pings against the app's configured warmup path receive a response with a 200 or 202 status code, organic traffic will be routed to the container. If a status code that is not in the list is received (such as 502), the platform will continue making pings until (1) a 200 or 202 is received, or (2) the container startup timeout limit is reached (see `WEBSITES_CONTAINER_START_TIME_LIMIT`). NOTE: If the container does not respond with an HTTP status code that is in the list, the platform will eventually fail the startup attempt and retry, which will result in 503 errors. ||

Reviewer: ```suggestion
| `WEBSITE_WARMUP_STATUSES` | A comma-delimited list of HTTP status codes that will be considered successful when the platform makes warmup pings against a newly started container. Used in conjunction with `WEBSITE_WARMUP_PATH`. By default, any status code is considered an indication that the container is ready for organic traffic. This appsetting can be used to require a specific response before organic traffic is routed to the container. `Example: 200,202`. If pings against the app's configured warmup path receive a response with a 200 or 202 status code, organic traffic will be routed to the container. If a status code that is not in the list is received (such as 502), the platform will continue making pings until (1) a 200 or 202 is received, or (2) the container startup timeout limit is reached (see `WEBSITES_CONTAINER_START_TIME_LIMIT`). Note that if the container doesn't respond with an HTTP status code that is in the list, the platform will eventually fail the startup attempt and retry, which will result in 503 errors. ||
```

| `WEBSITE_SCM_SEPARATE_STATUS` | Read-only. Shows whether the Kudu app is running in a separate process (`1`) or not (`0`). ||
| `WEBSITE_DNS_ATTEMPTS` | Number of times to try name resolve. ||
| `WEBSITE_DNS_TIMEOUT` | Number of seconds to wait for name resolve ||

> | `WEBSITES_CONTAINER_START_TIME_LIMIT` | The amount of time (in seconds) that the platform will wait for a container to become ready on startup. This settings applies for both code-based and container-based apps on App Service for Linux. The default value is `230`. When a container starts up, repeated pings are made against the container to gauge its readiness to serve organic traffic (see `WEBSITE_WARMUP_PATH` and `WEBSITE_WARMUP_STATUSES`). These pings are continuously made until either of the following is true: (1) a successful response is received, or (2) the start time limit is reached. If the container is not deemed ready within the configured timeout, the platform will fail the startup attempt and retry, which will result in 503 errors. For App Service for Windows Containers, the default start time limit is `10 mins`. You can change the start time limit by specifying a timespan like this `00:05:00` which indicates 5 minutes.||

Reviewer: ```suggestion
| `WEBSITES_CONTAINER_START_TIME_LIMIT` | The amount of time (in seconds) that the platform will wait for a container to become ready on startup. This setting applies to both code-based and container-based apps on App Service for Linux. The default value is `230`. When a container starts up, repeated pings are made against the container to gauge its readiness to serve organic traffic (see `WEBSITE_WARMUP_PATH` and `WEBSITE_WARMUP_STATUSES`). These pings are continuously made until either of the following is true: (1) a successful response is received, or (2) the start time limit is reached. If the container isn't deemed ready within the configured timeout, the platform will fail the startup attempt and retry, which will result in 503 errors. For App Service for Windows Containers, the default start time limit is `10 mins`. You can change the start time limit by specifying a timespan like this `00:05:00`, which indicates 5 minutes.||
```
